tasks/documentation.ts

```diff
@@ -27,13 +52,13 @@ export const generateDocumentation:( env:"development" | "production" ) => gulp.
 
 export const docsBuildDev:gulp.TaskFunction = gulp.series(
 	docsClean,
-	generateDocumentation("development")
+	generateDocumentation("development", descriptionTemplateRoute)
```

Comment:
It's not necessary to pass this parameter since is a value that not change.
Using a global variable is fine, while it is a constant value/variable since this not have the problem of changing the behavior.